### PR TITLE
 update descriptions to fix audit issues

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` ~~(after doing `brew install <formula>`)~~?

-----
`bwfmetaedit` copied from [`avimetaedit`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/avimetaedit.rb)

### CODE REVIEW

File: Formula/bwfmetaedit.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Bwfmetaedit < Formula
-  desc "Tool that supports embedding, validating, and exporting of metadata in Broadcast WAVE Format (BWF) files"
+  desc "Tool for embedding, validating, and exporting of BWF files metadata"
```
Comment (ilovezfs) [inline Formula/bwfmetaedit.rb]:
```
desc "Tool for embedding, validating, and exporting BWF file metadata"
```

Comment (commitay) [reply Formula/bwfmetaedit.rb]: updated

File: Formula/swagger-codegen.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class SwaggerCodegen < Formula
-  desc "Generation of API clients, server stubs, documentation from OpenAPI/Swagger definition"
+  desc "Generate clients, server stubs, documentation from OpenAPI spec"
```
Comment (ilovezfs) [inline Formula/swagger-codegen.rb]:
```
  desc "Generate clients, server stubs, and docs from an OpenAPI spec"
```

Comment (commitay) [reply Formula/swagger-codegen.rb]: updated

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
🛩

**Pull request merged**